.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["react", "es2015", "stage-0"]
```

Comment:
Do you really want stage-0?  This will allow using all the ES proposals from TC39, but will mean certain cases may be dropped by the standard after we start using them?

@KamranAsif  @jjwill any thoughts on what stage we want to take if any outside of what's already in the es2015 standard.  Note this isn't for LD but I think we should keep LD & future enterprise projects consistent except where LD has technical blockers.

Also I think you want to run react before es2015, so you should switch the ordering of the 2 (presets run last to first).